Formula/rmw.rb

```diff
@@ -0,0 +1,20 @@
+class Rmw < Formula
+  desc "OS portable cli trash can utility written in C"
+  homepage "https://github.com/andy5995/rmw/wiki"
+  url "https://github.com/andy5995/rmw/archive/v0.4.01.tar.gz"
```

Comment:
I'm the maintainer of rmw.

Thanks for submitting this @sexybiggetje 

This link my cause a problem. It redirects to a file with a different name (rmw-0.4.01.tar.gz).

If there's any problem, you can use this link instead:
https://github.com/andy5995/rmw/releases/download/v0.4.01/rmw-0.4.01.tar.gz

Reply:
The link is as used in the Formula cookbook, and local installation from that link works fine.
Wonder why Jenkins fails below tho.